### ISSUE TITLE
#212 breaks MVP

### ISSUE THREAD

**Issue by RubenJacobse** [TUDelft-CNS-ATM/bluesky#213]
#212 causes crashes in MVP when the distance condition is not met and `dv1` and `dv2` are left unassigned.

```python
    # If intruder is outside the ownship PZ, then apply extra factor
    # to make sure that resolution does not graze IPZ
    if asas.Rm<dist and dabsH<dist:
        # Compute the resolution velocity vector in horizontal direction.
        # abs(tcpa) because it bcomes negative during intrusion.
        erratum=np.cos(np.arcsin(asas.Rm/dist)-np.arcsin(dabsH/dist))
        dv1 = ((asas.Rm/erratum - dabsH)*dcpa[0])/(abs(tcpa)*dabsH)
        dv2 = ((asas.Rm/erratum - dabsH)*dcpa[1])/(abs(tcpa)*dabsH)


```

I suggest adding an `else` block which uses the original definitions of dv1 and dv2 in case of intrusion:
```python
    # If intruder is outside the ownship PZ, then apply extra factor
    # to make sure that resolution does not graze PZ
    if asas.Rm < dist and dabsH < dist:
        # Compute the resolution velocity vector in horizontal direction
        # abs(tcpa) because it becomes negative during intrusion
        erratum = np.cos(np.arcsin(asas.Rm/dist) - np.arcsin(dabsH/dist))
        dv1 = ((asas.Rm / erratum - dabsH) * dcpa[0]) / (abs(tcpa) * dabsH)
        dv2 = ((asas.Rm / erratum - dabsH) * dcpa[1]) / (abs(tcpa) * dabsH)
    else:
        dv1 = (iH * dcpa[0]) / (abs(tcpa) * dabsH)
        dv2 = (iH * dcpa[1]) / (abs(tcpa) * dabsH)

```

**Comment by wschaberg** [comment]
Hi Ruben!

You're completely right, I forgot to include the exception. Thanks for the heads up!

**Comment by RubenJacobse** [comment]
Issue fixed in #214
